raphtory/src/db/internal/core_ops.rs

```diff
@@ -99,19 +102,16 @@ impl<const N: usize> CoreGraphOps for InnerTemporalGraph<N> {
 
     fn static_edge_prop(&self, e: EdgeRef, name: &str) -> Option<Prop> {
```

Comment:
This is indeed very broken, we need to figure out merging properties across layers!